study.md

```diff
@@ -44,21 +52,29 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+<!-- I'm  not very sure what are meant by views, but I think it's data
+associated with the current site in-view
+Running out of time on this one as I need to go beyond two hours of study time
+to finish this question  -->
```

Comment:
Views are often though of as chunks of HTML that may or may not be visible at any given time while visiting a web site.